### ISSUE TITLE
Issue with tRNA prediction module

### ISSUE THREAD

**Issue by EdderDaniel** [oschwengers/bakta#150]
I installed Bakta through conda (latest version) and I got this error while trying to annotate some bacterial genomes. It only happened in like 6 genomes out of a db of 500, so not really a big problem, but I was wondering what could be wrong with those genomes that triggered this

parse genome sequences...
        imported: 255
        filtered & revised: 255
        contigs: 255

start annotation...
predict tRNAs...
Traceback (most recent call last):
  File "/home/bostosed/miniconda3/bin/bakta", line 10, in <module>
    sys.exit(main())
  File "/home/bostosed/miniconda3/lib/python3.8/site-packages/bakta/main.py", line 150, in main
    genome['features'][bc.FEATURE_T_RNA] = t_rna.predict_t_rnas(genome, contigs_path)
  File "/home/bostosed/miniconda3/lib/python3.8/site-packages/bakta/features/t_rna.py", line 123, in predict_t_rnas
    assert trna['nt'] == nt
AssertionError

**Comment by oschwengers** [comment]
Thanks @EdderDaniel, 
thanks for reporting - very interesting! Could you provide a genome or a single contig to reproduce and debug this?

**Comment by EdderDaniel** [comment]
[GCA_0026083451.zip](https://github.com/oschwengers/bakta/files/9633605/GCA_0026083451.zip)

Sure! Here is one of the genomes that had the issue

**Comment by oschwengers** [comment]
Thanks for the test genome. It contains 65 instances of an `W` - the IUPAC ambiguity code for either `A` or `T`.
However, tRNAscan-se writes `S`to its fasta output - the IUPAC ambiguity code for either `G` or `C`.

Here's an example of Ala_trna on contig contig_226 -[3733..3808]:
```
genome-nt   = GGGGGTGTAGCTCAGCTGGGAGAGCGCYTGCWTKGCAWGCARGAKGTCRTCGGTTCGATCCCGTTYACCTCCACCA
trnascan-nt = GGGGGTGTAGCTCAGCTGGGAGAGCGCYTGCSTKGCASGCARGAKGTCRTCGGTTCGATCCCGTTYACCTCCACCA
```

Bakta extracts the `nt` sequence of each feature from the raw genome sequence and due to the fact that this has not occurred on thousands of genomes before, this is a very rare observation - good catch!
You might want to report this upstream to the [tRNAscan-SE](https://github.com/UCSC-LoweLab/tRNAscan-SE) developers. I've removed the `assert` on line 123 since it was just fine for all former genomes so far.

Again, thanks a lot!

**Comment by oschwengers** [comment]
FYI: This bug is fixed by tRNAscan-SE release [v2.0.11](https://github.com/UCSC-LoweLab/tRNAscan-SE/releases/tag/v2.0.11)
